src/cockpit/packages.py

```diff
@@ -83,19 +84,27 @@ def parse_accept_language(accept_language: str) -> List[str]:
     locales = []
     for language in accept_languages:
         language = language.strip()
-        locale, _, weight = language.partition(';q=')
-        weight = float(weight or 1)
+        locale, _, weightstr = language.partition(';q=')
```

Comment:
You really don't like variable aliasing, do you :grin:

Reply:
mypy doesn't like a variable name within a given scope to have two different types.